src/k8s/components/indexed-search/generate/service/indexed-search.dhall

```diff
@@ -49,9 +49,8 @@ let Service/generate
                 , name = Some "indexed-search"
                 }
               , spec = Some Kubernetes/ServiceSpec::{
-                , ports = Some
-                  [ Kubernetes/ServicePort::{ name = Some "port", port = 6070 }
-                  ]
+                , clusterIP = Some "None"
+                , ports = Some [ Kubernetes/ServicePort::{ port = 6070 } ]
```

Comment:
the `name` also doesn't appear in the diff

Reply:
cool